src/test/java/com/datadoghq/sketch/ddsketch/SerializerTest.java

```diff
@@ -0,0 +1,93 @@
+package com.datadoghq.sketch.ddsketch;
+
+import com.datadoghq.sketch.ddsketch.footprint.Distribution;
+
+import com.datadoghq.sketch.ddsketch.mapping.*;
+import com.datadoghq.sketch.ddsketch.store.*;
+import com.datadoghq.sketch.ddsketch.store.Store;
+import com.google.protobuf.InvalidProtocolBufferException;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.nio.ByteBuffer;
+import java.util.Iterator;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import static com.datadoghq.sketch.ddsketch.footprint.Distributions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class SerializerTest {
+
+    public static Stream<Arguments> sketches() {
+        return Stream.of(
+                new LogarithmicMapping(0.01),
+                new BitwiseLinearlyInterpolatedMapping(0.01),
+                new LinearlyInterpolatedMapping(0.01),
+                new QuadraticallyInterpolatedMapping(0.01),
+                new CubicallyInterpolatedMapping(0.01)
+        )
+                .flatMap(im -> Stream.<Supplier<Store>>of(
+                        SparseStore::new,
+                        () -> new CollapsingLowestDenseStore(1000),
+                        () -> new CollapsingHighestDenseStore(1000),
+                        UnboundedSizeDenseStore::new,
+                        PaginatedStore::new
+                ).map(s -> (Supplier<DDSketch>) () -> new DDSketch(im, s)))
+                .flatMap(supplier -> Stream.of(
+                        NORMAL.of(0, 1),
+                        NORMAL.of(100, 10),
+                        POISSON.of(0.01),
+                        POISSON.of(0.99),
+                        POINT.of(42),
+                        UNIFORM.of(100)
+                ).map(dist -> Arguments.of(supplier, dist)));
+    }
+
+
+    @ParameterizedTest
+    @MethodSource("sketches")
+    public void testProtobufSerialization(Supplier<DDSketch> sketchSupplier, Distribution distribution) throws InvalidProtocolBufferException {
+        DDSketch sketch = load(sketchSupplier, distribution);
+        ByteBuffer buffer = sketch.serialize();
+        assertEquals(DDSketchProtoBinding.toProto(sketch).getSerializedSize(), buffer.remaining());
+
+        assertArrayEquals(DDSketchProtoBinding.toProto(sketch).toByteArray(), buffer.array());
+        com.datadoghq.sketch.ddsketch.proto.DDSketch proto =
+                com.datadoghq.sketch.ddsketch.proto.DDSketch.parseFrom(buffer);
+
+        DDSketch recovered = DDSketchProtoBinding.fromProto(UnboundedSizeDenseStore::new, proto);
+        assertEquals(sketch.getCount(), recovered.getCount(), 1e-7);
```

Comment:
We use `AccuracyTester.FLOATING_POINT_ACCEPTABLE_ERROR` in `DDSketchTest` and `StoreTest`.